mLRS/Common/sx-drivers/sx126x_driver.h

```diff
@@ -250,7 +250,11 @@ class Sx126xDriverCommon : public Sx126xDriverBase
         SetRxGain(SX126X_RX_GAIN_BOOSTED_GAIN);
         SetOverCurrentProtection(SX126X_OCP_CONFIGURATION_140_MA); // default for SX1262 according to data sheet, but can't hurt
 
+        #if defined POWER_PA_SE2435L || defined POWER_PA_MATEK_MR900_30 
```

Comment:
I guess I would rather like a generic define such as "POWER_USE_PA_CONFIG_10_DBM"
to not spill the generic drivers with device specifics

Reply:
Fair enough - new define + moved to hal-power